app/app.ts

```diff
@@ -0,0 +1,23 @@
+import { createRequestHandler } from "@react-router/express";
+import express from "express";
+import "react-router";
+
+declare module "react-router" {
+  interface AppLoadContext {
+
+  }
+}
+
+export const app = express();
+
+app.use((req, res, next) => {
+  if (req.path.startsWith('/socket.io/')) {
+    return next();
```

Comment:
I don't know what would be next. I don't see how the socket server is attached to the web server

Reply:
next is a way to be able to push forward to the next route that matches /socket.io/ . The socket.io library automagically binds this to our express server